src/client/Microsoft.Identity.Client/AuthScheme/Bearer/BearerAuthenticationScheme.cs

```diff
@@ -10,6 +10,8 @@ internal class BearerAuthenticationScheme : IAuthenticationScheme
     {
         internal const string BearerTokenType = "bearer";
 
+        public int TelemetryId => 1;
```

Comment:
I would create an enum like we use [CacheRefreshReason](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/Cache/CacheRefreshReason.cs). I think this makes the code more readable.

Something like:
```csharp
internal enum TelemetryTokenType // or TokenType
{
Bearer = 1,
Pop = 2,
SshCert = 3,
PopExtensibilityApi = 4,
}
```

Reply:
I was thinking of using an enum earlier but here the scenario is different. We don't have to evaluate the value it simply depends on which type of auth scheme object is created and hence used this,

Reply:
imo it helps eliminate scenarios like this where we have to do extra explanation of what the numbers are - with enums, the code is self documenting.
```csharp
// Token type is to indicate 1 - bearer, 2 - pop, 3 - ssh-cert, 4 - external.
sb.Append(eventInProgress.TokenType);
```
Similar thing like in tests: 
instead of
```csharp
private readonly string XClientCurrentTelemetryROPCFailure = $"{TelemetryConstants.HttpTelemetrySchemaVersion}|1003,{CacheRefreshReason.NotApplicable:D},,,|0,1,1";
```
can be this
```csharp
private readonly string XClientCurrentTelemetryROPCFailure = $"{TelemetryConstants.HttpTelemetrySchemaVersion}|1003,{CacheRefreshReason.NotApplicable:D},,,|0,1,{TokenType.Bearer}";
```

Reply:
Makes sense. I will update it.